src/main/java/org/jabref/model/entry/field/StandardField.java

```diff
@@ -25,6 +25,7 @@ public enum StandardField implements Field {
     ARCHIVEPREFIX("archiveprefix"),
     ASSIGNEE("assignee", FieldProperty.PERSON_NAMES),
     AUTHOR("author", FieldProperty.PERSON_NAMES),
+    ARXIVID("arXivId", FieldProperty.VERBATIM),
```

Comment:
Arxiv is no StandardField, you need to use eprint for this
see chapter 3.14.7 Electronic Publishing Information of the biblatex spec
 http://mirrors.ctan.org/macros/latex/contrib/biblatex/doc/biblatex.pdf